### PR TITLE
Fix a couple minor urgent cosmetic issues in OmniNav.

### PR DESCRIPTION
**Pull request by kaylaziegler**
Story: https://trello.com/c/3I5rdmv3

Issue #289

### CODE REVIEW

File: app/assets/stylesheets/omni_nav/omni_nav_v2/omni-nav.scss [hunk 1]
```diff
@@ -588,6 +589,7 @@ html.omni-nav-v2 {
           & > a {
             text-transform: uppercase;
             font-size: 15px;
+            white-space: nowrap;
```
Comment (kaylaziegler) [inline app/assets/stylesheets/omni_nav/omni_nav_v2/omni-nav.scss]: Added this `white-space: nowrap` rule to the 2 areas where users reported the weirdly wrapped text on zoomed-in screens.

**Comment by kaylaziegler** [comment]
Tested on IE, Firefox, Chrome and Safari. I wasn't able to reproduce the font issue reported by Robin Pendergraft but I implemented the syntax that Wai recommended for that font-family

File: app/assets/stylesheets/omni_nav/omni_nav_v2/omni-nav.scss [hunk 2]
```diff
@@ -887,7 +889,7 @@ html.omni-nav-v2 {
       padding: 1px 0 0 20px;
       border-left: 1px solid #2f2f2f;
       font-size: 18px;
-      font-family: Minion Pro;
+      font-family: 'minion-pro';
```
Comment (tatwell) [inline app/assets/stylesheets/omni_nav/omni_nav_v2/omni-nav.scss]: @waidelacruz Shouldn't there be some fallback fonts here?

    font-family: minion-pro, serif;

File: app/views/omni_nav_v2/_global_nav.html.erb [hunk 3]
```diff
@@ -58,6 +57,18 @@
...
+        <div class="global-nav-dropdown">
+          <ul>
+            <li><a class="icon icon-medal" href="research/sponsored-projects-services/index.aspx">Sponsored Projects Services</a></li>
+            <li><a class="icon icon-clipboard5" href="research/integrity/index.aspx">Research Integrity</a></li>
+            <li><a class="icon icon-library4" href="research/institutes-and-centers/index.aspx">Institutes and Centers</a></li>
+            <li><a class="icon icon-lamp8" href="research/office-undergraduate-research-creative-activity/index.aspx">Undergraduate Research and Creative Activity</a></li>
+            <li><a class="icon icon-microscope" href="research/graduate-research/index.aspx">Graduate Research Support</a></li>
+          </ul>
+        </div>
+      </li>
```
Comment (tatwell) [inline app/views/omni_nav_v2/_global_nav.html.erb]: Was this change request added to the story by a product owner?

Comment (kaylaziegler) [reply app/views/omni_nav_v2/_global_nav.html.erb]: No, it just didn't reflect the either one of the actual global navs and I wanted the text to at least match the screenshots that were reported by the users for this text wrapping issue.

Comment (tatwell) [reply app/views/omni_nav_v2/_global_nav.html.erb]: Oh, I see. This is just the Rails app sample. Disregard my original comment.

File: app/assets/stylesheets/omni_nav/omni_nav_v2/omni-nav.scss (hunk 2 above)
Comment (tatwell) [reply app/assets/stylesheets/omni_nav/omni_nav_v2/omni-nav.scss]: If serif is inherited somewhere else, then my question can be disregarded.

File: app/views/omni_nav_v2/_global_nav.html.erb (hunk 3 above)
Comment (mandythomas) [reply app/views/omni_nav_v2/_global_nav.html.erb]: @kaylaziegler Meagan Sawyer (new person in UA) would be a good person to ask to test this. She was seeing the odd rendering of the omniNav menu items where the initial letter was a superscript. She didn't uninstall the Minion  Pro font like the other person I'd asked to try that so would still see it broken on prod and fixed on dev. You should email her (or Ross) to ask her to test for you.

Comment (kaylaziegler) [reply app/views/omni_nav_v2/_global_nav.html.erb]: @mandythomas Thank you for letting me know about Megan, I had her test the staging link that I sent in the Review Request email and it fixed the problem for her! very exciting

**Pull request merged**